tooling/cli.rs/config_definition.rs

```diff
@@ -292,6 +295,9 @@ pub struct WindowConfig {
   /// Whether the window starts as fullscreen or not.
   #[serde(default)]
   pub fullscreen: bool,
+  /// Whether the window will be initially hidden or focused.
+  #[serde(default = "default_focus")]
+  pub focus: bool,
```

Comment:
Changing this in tauri.conf.json doesn't seem to make a difference (at least on Windows).

Reply:
We decided to remove `focus` from `WindowBuilder` and `WindowAttributes` at Tao, so if tauri wants to support this, they would have to explicitly call `window.set_focus()`.

Reply:
Should i remove the focus stuff again? -> Make this pr only about the center thingy?

Reply:
It is @lucasfernog decision.